src/pages/PageNotFound.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+export const PageNotFound: React.FC = () => {
+  return (
+    <div
+      style={{
```

Comment:
Every new component we create should have its own folder containing three files:
- the tsx component file
- the css module file for styling
- the test file

This provides the best flexibility for future expansion and ensures we have at least 1 test for every component.